sdk/include/xdk/winnt_old.h

```diff
@@ -2560,9 +2560,13 @@ typedef struct _RTL_CRITICAL_SECTION_DEBUG {
   LIST_ENTRY ProcessLocksList;
   DWORD EntryCount;
   DWORD ContentionCount;
+#if 1  
+  DWORD Spare[2];
+#else /* neither Wine, no ReactOS use this fields (they are in 0x600+ struct definition) */
   DWORD Flags;
```

Comment:
This definition is the correct one that matches PSDK. As previous discussed, that is what our headers should provide by default.
You can use `__WINESRC__` to allow Wine code access to the other version.

Reply:
tryed to use __WINESRC__ to exact files as you said in 1190 but this triggered errors in headers (setupapi even have explicit "#undef __WINESRC__" ). With _WINE it's run ok (checked vbox guest additions, firefox and .net 4)

Reply:
I don't like the idea of using `_WINE` because Wine does not use it to make this distinction.
We also shouldn't be defining `__WINESRC__` (or `_WINE`) in files that aren't Wine-synced.
I would suggest something like https://github.com/ThFabba/reactos/commit/c0ceb3f0f034be737c7f9e47829d3048003ffb0d instead.